src/methods/modular_programs/easy/q7/MainTest.java

```diff
@@ -0,0 +1,86 @@
+package methods.modular_programs.easy.q7;
+
+import global.BaseTest;
+import global.tools.CustomAssertions;
+import global.tools.TestOption;
+import global.utils.MethodUtil;
+import global.variables.Clause;
+import global.variables.clauses.DoubleLiteral;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
+    // Parsons
+    public Clause[] testSentence() {
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = "5.23 17.95";
+        return new Clause[]{
+                new StringLiteral("Enter the first number: "),
+                new NewLine(),
+                new StringLiteral("Enter the second number: "),
+                new NewLine(),
+                new StringLiteral("The value of the first variable raised to the second variable is "),
+                new DoubleLiteral("raised"),
+                new NewLine(),
+                new StringLiteral("The value of the greater variable converted from degrees to radians is "),
+                new DoubleLiteral("radians")
+        };
+    }
+
+    public void runMain() {
+        TwoMethods.main(new String[0]);
+    }
+
+    static Stream<Arguments> raisedCalcInputProvider() {
+        return Stream.of(
+                Arguments.of(2, 4, 16.0),
+                Arguments.of(3, 3, 27.0),
+                Arguments.of(8.3, 11.2, 1.966412537842738E10),
+                Arguments.of(-7.2, 3, -373.24800000000005),
+                Arguments.of(-3.6, 4, 167.9616),
+                Arguments.of(5.67, -9.6, 5.8291151506320536E-8));
+    }
+
+    static Stream<Arguments> degreeCalcInputProvider() {
+        return Stream.of(
+                Arguments.of(2, 4, 0.06981317007977318),
+                Arguments.of(3, 3, 0.05235987755982989),
+                Arguments.of(8.3, 11.2, 0.1954768762233649),
+                Arguments.of(-7.2, 3, 0.05235987755982989),
+                Arguments.of(-3.6, 4, 0.06981317007977318),
+                Arguments.of(5.67, -9.6, 0.09896016858807849));
```

Comment:
Same thing here, there should be a 0 case

Reply:
Also, you have the second number as the larger one most of the time, so just be careful about that